mitreattack/collections/README.md

```diff
@@ -24,4 +25,31 @@ with open('collection_index.json', 'r') as input_file:
         generated_md = IndexToMarkdown.index_to_markdown(input_index)  # Convert index to markdown
         output_file.write(generated_md)
 print(generated_md)
-```
\ No newline at end of file
+```
+
+## collection_to_index.py
+collection_to_index.py provides the CollectionToIndex class, which proves a means by which to summarize existing [collections](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/blob/master/docs/collections.md#collections)
+into a single [collection index](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/blob/master/docs/collections.md#collection-indexes) file.
+The CollectionToIndex class contains the generate_index function, which when provided with a name, description, root url (pointing to where the raw collections are stored), 
+and a list of either files, folders, or already loaded bundles in the form of dictionaries, will create a summarizing index.
+#### Example Usage
+```python
+import json
+from mitreattack.collections import CollectionToIndex
+
+output_indexA = CollectionToIndex.generate_index(name='example', description='example index', 
+                                                 root_url='www.example.com', 
+                                                 files=['/path/to/collection1.json', '/path/to/collection2.json'], 
+                                                 folders=None, bundles=None)
+output_indexB = CollectionToIndex.generate_index(name='example2', description='demonstration index',
+                                                 root_url='www.example.com',
+                                                 files=None, folders=['/path/to/folder/with/collections'], bundles=None)
+with open('path/to/bundle/bundleC.json', 'r') as f:
+    data = json.load(f)
+output_indexC = CollectionToIndex.generate_index(name='example3', description='exhibit index',
+                                                 root_url='www.example.com',
+                                                 files=None, folders=None, bundles=[data])
```

Comment:
Documentation here still uses `bundles` instead of `sets`.

Reply:
Woops, I'll fix that.